src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -59,6 +59,7 @@ sap.ui.define([
 	 *            (oResource.meta.profile[0]) at the requested resource
 	 * @param {string} [mParameters.defaultSubmitMode] The default SubmitMode for all bindings which are associated with this model
 	 * @param {string} [mParameters.defaultFullUrlType='uuid'] The default FullUrlType if the default submit mode is either batch or transaction
+	 * @param {array}  [mParameters.defaultQueryParameters=[]] The default query parameters to be passed to for all GET requests if there is no specific binding info. It should be of type key:value pairs in the array E.g. {'_total':'accurate'} -> http://hl7.org/fhir/http.html#parameters
```

Comment:
I would recommend using `@param {object[]} [mParameters.defaultQueryParameters=[]]`

Reply:
The default query parameters to be passed to for all GET requests

This part I understand. However the part "if there is no specific binding info". I don't understand. Do you mean. The query parameter can be override by binding specific parameters?

Reply:
Right now we are using the configuration only for GET ALL .. Have updated the JSdoc..